services/apps/alcs/src/providers/typeorm/migrations/1689187139689-obsolete_nullable_status_code.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class obsoleteNullableStatusCode1689187139689
+  implements MigrationInterface
+{
+  name = 'obsoleteNullableStatusCode1689187139689';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
Why are we making a column not nullable and then dropping it? Could we just drop the column?

Reply:
That is iterative local development. But yes, I'll leave delete only in final version.